engine/swarm/admin_guide.md

```diff
@@ -251,8 +237,85 @@ Before you forcefully remove a manager node, you must first demote it to the
 worker role. Make sure that you always have an odd number of manager nodes if
 you demote or remove a manager
 
+## Back up the swarm
+
+Docker manager nodes store the swarm state and manager logs in the
+`/var/lib/docker/swarm/` directory. In 1.13 and higher, this data includes the
+keys used to encrypt the Raft logs. Without these keys, you will not be able
+to restore the swarm.
+
+You can back up the swarm using any manager. Use the following procedure.
+
+1.  If the swarm has auto-lock enabled, you will need the unlock key in order
+    to restore the swarm from backup. Retrieve the unlock key if necessary and
+    store it in a safe location. If you are unsure, read
+    [Lock your swarm to protect its encryption key](/engine/swarm/swarm_manager_locking.md).
+
+2.  Stop Docker on the manager before backing up the data, so that no data is
+    being changed during the backup. It is possible to take a backup while the
+    manager is running (a "hot" backup), but this is not recommended and your
+    results will be less predictable when restoring. While the manager is down,
+    other nodes will continue generating swarm data that will not be part of
+    this backup.
+
+    > **Note**: Be sure to maintain the quorum of swarm managers. For instance,
+    > if you normally run with three managers, you may need to add a manager
+    > before you can stop one, in case a running manager becomes unavailable
+    > during the backup. A two-manager swarm will lose quorum if one manager
+    > becomes unavailable.
+
+3.  Back up the entire `/var/lib/docker/swarm` directory.
+
+4.  Restart the manager.
+
+To restore, see [Restore from a backup](#restore-from-a-backup).
+
 ## Recover from disaster
 
+### Restore from a backup
+
+After backing up the swarm as described in
+[Backing up the swarm](#backing-up-the-swarm), use the following procedure to
+restore the data to a new swarm.
+
+1.  Shut down Docker on the target host machine where the swarm will be restored.
+
+3.  Remove the contents of the `/var/lib/docker/swarm` directory on the new
+    swarm.
+
+4.  Restore the `/var/lib/docker/swarm` directory with the contents of the
+    backup.
+
+    > **Note**: The new node will use the same encryption key for on-disk
+    > storage as the old one. It is not possible to change the on-disk storage
+    > encryption keys at this time.
+    >
+    > In the case of a swarm with auto-lock enabled, the unlock key is also the
+    > same as on the the old swarm, and the unlock key will be needed to
+    > restore.
+
+5.  Start Docker on the new node. Unlock the swarm if necessary. Re-initialize
+    the swarm using the following command, so that old nodes do not attempt to
+    reconnect to the swarm.
```

Comment:
It's more so that this node doesn't attempt to connect to old nodes. If you had other nodes still running, presumably you wouldn't need to use the backup.